### ISSUE TITLE
FR : Extend 'Send feedback after control'

### ISSUE THREAD

**Issue by DaveInDev** [helgoboss/realearn#711]
As discussed on the forum (I'm "binbinhfr" in repear's forum), it would be nice to implement this feature, that would help on controllers like Behringer's which change momentary the button's LED color everytime you press them...
Anytime Realearn triggers a control on a specified virtual button, Realearn could verify any feedback-only mappings containing the same virtual button, and trigger these feedbacks again, to refresh the faulty LED's status.
I could be an option appearing in feedback option list of the mapping (like "retrigger on other mapping control change") ?
Notice that, before retrigering the feedback, and reseting the LED at its correct value, I suppose that Realearn should wait to receive the final "note off" (= release pad) of the button, that really mark the end of the lit/unlit momentary effect...

PS : Benjamin, I swear I will test this feature if you implement it. :-)

**Comment by DaveInDev** [comment]
Hi helgoboss, 
Any chance that this request will be implemented one day ?  I'm still around, and interested  ;-)

**Comment by helgoboss** [comment]
Yes, but emphasis is on _one day_ ;) I have a full-time contract at the moment so it's a bit hard and there are multiple other FRs in the pipeline as well.

**Comment by DaveInDev** [comment]
I perfectly understand. No hurry.   :-)
But as you are often saying on the forum that users are not keeping in touch, so I just wanted to show that I'm still around, using realearn, and still interested ;-)
Have a nice weekend.

**Comment by helgoboss** [comment]
Ah thank you, yes, definitely nice to see the continued interest :) Asking from time to time definitely helps.

**Comment by vonglan** [comment]
@DaveInDev : I don't understand your use case from the description here. Can you post a link here to the forum discussion? (I have the related issue #777 .)

**Comment by DaveInDev** [comment]
@vonglan reading yur issue, it seems that we have the same problem concerning the Behringer xtouch mini behaviour, that changes the LED status by itself. I did not find any solution for the moment, I'm still waiting for @helgoboss to implement this option.
